machine/v1beta1/types_awsprovider.go

```diff
@@ -91,6 +91,9 @@ type AWSMachineProviderConfig struct {
 	// +kubebuilder:validation:Maximum:=7
 	// +optional
 	PlacementGroupPartition *int32 `json:"placementGroupPartition,omitempty"`
+	// CapacityReservationID specifies the target Capacity Reservation into which the instance should be launched.
```

Comment:
Comment should start with the json tag version of the field name.

What is the expected format of this field? Can you explain that in the godoc for the field?

Reply:
Updated the comments..

Reply:
/cc @JoelSpeed